packages/components/src/utils.ts

```diff
@@ -1,13 +1,15 @@
 import { createIntl } from 'react-intl'
-import snarkdown from 'snarkdown'
+import MarkdownIt from 'markdown-it'
 import { MessageConfig } from './typings'
 
-export const renderUnsafeHTML = (message: string = '', escaped: boolean): string => {
+export const markdownToHtml = (message: string = '', escaped: boolean): string => {
   if (escaped) {
     message = message.replace(/</g, '&lt;').replace(/>/g, '&gt;')
   }
 
-  const html = snarkdown(message)
+  const md = new MarkdownIt({ linkify: true })
```

Comment:
**note**: The bug was that the webchat integration sends text messages with markdown=true always (by design, so that people can freely use markdown in the Studio), so to fix the issue we need to convert plain URLs in markdown to actual links.